cli/command/registry.go

```diff
@@ -28,16 +28,22 @@ func ElectAuthServer(ctx context.Context, cli Cli) string {
 	// used. This is essential in cross-platforms environment, where for
 	// example a Linux client might be interacting with a Windows daemon, hence
```

Comment:
Looks like this was added in https://github.com/moby/moby/pull/19891 at a time where a temporary/special Windows registry was set up: https://github.com/moby/moby/issues/18019

Actually thinking if that handling is now just dead code and no longer needed